src/components/ShareBlock/EmbedPreview.style.js

```diff
@@ -35,7 +35,7 @@ export const CenteredModal = styled(Modal)`
 
 export const EmbedPreviewScrollContainer = styled(Box)`
   max-height: 95vh;
-  overflow-y: hidden;
+  overflow-y: scroll;
```

Comment:
Would check with @mikelehen ; was able to show unwanted scroll bars in his view

Reply:
Looks okay to me.

FWIW- To duplicate my config, you can either plug a mouse into your macbook, use Windows, or [set this option in OSX](https://osxdaily.com/2011/08/03/show-scroll-bars-mac-os-x-lion/).

Reply:
Ah ok. I had changed it to scroll because when the embed preview + the snippets are stacked in a column and the browser is short, the snippets aren't accessible. But I guess if this wasn't of concern before, I can keep it as it was if that's preferred

Reply:
I think as long as it's not causing problems, your change is fine.  Though I'm not sure what "when the browser is short" means, exactly.  As long as it's embedded as an iframe with a fixed width / height, can we assume that won't be an issue?

For context, the issue we had previously was that for some reason, the content would always be _slightly_ too big for the container (by a pixel or something) and so you'd end up with a dummy scrollbar all the time.

Reply:
Got it. Yeah, this is the wrapper for the entire preview modal that opens when you first click "embed on your website" in the CAN footer (the entire white modal below), not for the iframe itself